### PR TITLE
adding release notes for 0.18.13

### PR DESCRIPTION
**Pull request by alexwykoff**
Audtiors : @bbondy @bsclifton

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,36 @@
...
+ - Fixed extension tabs not auto closing when finished. ([#9132](https://github.com/brave/browser-laptop/issues/9132))
+ - Fixed Pocket and LastPass home page loads after logging into account. ([#9117](https://github.com/brave/browser-laptop/issues/9117))
+ - Fixed TLD of long domain display in title tooltip on bravery panel. ([#9089](https://github.com/brave/browser-laptop/issues/9089))
+ - Fixed about:brave information display for Windows. ([#8933](https://github.com/brave/browser-laptop/issues/8933))
+ - Fixed bookmarks in Other Bookmarks folder from being synced to the bookmark toolbar. ([#8024](https://github.com/brave/browser-laptop/issues/8024))
+ - Fixed adding a bookmarks folder makes the browser unresponsive in Sync. ([#9745](https://github.com/brave/browser-laptop/issues/9745))
+ - Fixed bookmark deletion may lead to duplicated bookmarks in Sync. ([#9724](https://github.com/brave/browser-laptop/issues/9724))
+ - Fixed blank tabs bar when increasing the number of tabs per tab set. ([#7806](https://github.com/brave/browser-laptop/issues/7806))
+ - Updated to Chromium 60. ([#9947](https://github.com/brave/browser-laptop/issues/9947))
```
Comment (bbondy) [inline CHANGELOG.md]: Please put the full Chromium version here.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,38 @@
 # Changelog
 
+## [0.18.13](https://github.com/brave/browser-laptop/releases/tag/v0.18.13dev)
+ - Added a welcome screen for first time users. ([#9423](https://github.com/brave/browser-laptop/issues/9423))
+ - Added Chromium spell checker. ([#9880](https://github.com/brave/browser-laptop/issues/9880))
```
Comment (bsclifton) [inline CHANGELOG.md]: maybe it would be good to call out "(in place of Electron spell checker)"

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,38 @@
...
+ - Added an option to disable "Top Sites" in autocomplete. ([#4977](https://github.com/brave/browser-laptop/issues/4977))
+ - Added "Hide Lower" button on Brave Payments. ([#9137](https://github.com/brave/browser-laptop/issues/9137))
+ - Optimized sending messages only to needed windows instead of all windows. ([#9701](https://github.com/brave/browser-laptop/issues/9701))
+ - Fixed showing the URL in the location bar for invalid URLs (instead of about:error). ([#2025](https://github.com/brave/browser-laptop/issues/2025)) 
+ - Fixed opening a context menu freezes video (ex: Youtube). ([#5470](https://github.com/brave/browser-laptop/issues/5470))
```
Comment (NejcZdovc) [inline CHANGELOG.md]: @bsclifton @bbondy in which PR did we fix this. I think that async menu was already implemented in Jun,probably in 0.16.x

Comment (bsclifton) [inline CHANGELOG.md]: good catch- this was def 0.16.x 😄

**Comment by alexwykoff** [comment]
updated, left 5470 in there because it was tied to pr https://github.com/brave/browser-laptop/pull/9128

**Comment by bsclifton** [comment]
@alexwykoff whoops- I think that was a miss on our part. That was released with Chromium 59 actually, which means it was in 0.16.x.  I'll update the milestone for the issue and the PR to properly be 0.16.x.  After removing that, I think we're good for a merge 😄

**Comment by bsclifton** [comment]
Removed the async menu option from the releases page and from the notes.  Also squashed commits

@alexwykoff in the future, would you PR against master? This way it's similar to other PRs (where you merge then cherry-pick back until the selected milestone)

**Pull request merged**